pom.xml

```diff
@@ -85,13 +91,22 @@
 	</dependencies>
 	<build>
 		<plugins>
+			<plugin>
+				<groupId>org.apache.maven.plugins</groupId>
+				<artifactId>maven-surefire-plugin</artifactId>
+				<version>${maven-surefire-plugin.version}</version>
+				<configuration>
+					<reuseForks>true</reuseForks>
+				</configuration>
+			</plugin>
 			<plugin>
 				<groupId>org.jacoco</groupId>
 				<artifactId>jacoco-maven-plugin</artifactId>
 				<configuration>
 					<excludes>
 						<exclude>**/se/sundsvall/disturbance/Application*</exclude>
 						<exclude>**/generated/se/sundsvall/**/*</exclude>
+						<exclude>**/se/sundsvall/disturbance/integration/db/model/*Entity_**</exclude>
```

Comment:
Why?

Reply:
Generated entities. Note the `_`

Reply:
Because `JPAModelGen` generates these *Entity_ classes that only contains 
`public static volatile SingularAttribute`s and `public static final String`s

e.g 	`public static volatile SingularAttribute<StakeholderEntity, Long> id;`
	`public static final String ID = "id";`

Reply:
What Linus said :)